airflow/serialization/serialized_objects.py

```diff
@@ -822,7 +822,9 @@ def _serialize_node(cls, op: BaseOperator | MappedOperator, include_deps: bool)
 
         if op.operator_extra_links:
             serialize_op["_operator_extra_links"] = cls._serialize_operator_extra_links(
-                op.operator_extra_links
+                op.operator_extra_links.fget(op)  # type: ignore
```

Comment:
Better to use `__get__` here if possible, I think that’s what Mypy is trying to persuade you into. (And to repeat myself, please try to not ignore things until you’ve exhausted all possibilities)